NineChronicles.Headless/GraphTypes/StandaloneQuery.cs

```diff
@@ -51,8 +52,7 @@ public StandaloneQuery(StandaloneContext standaloneContext, IConfiguration confi
                     }
 
                     return new StateContext(
-                        chain.ToAccountStateGetter(blockHash),
-                        chain.ToAccountBalanceGetter(blockHash),
+                        chain.GetBlockState(blockHash),
```

Comment:
Since below statement is on the above,
```c#
BlockHash? blockHash = context.GetArgument<byte[]>("hash") switch
{
    byte[] bytes => new BlockHash(bytes),
    null => standaloneContext.BlockChain?.Tip?.Hash,
};
```
Null `hash` argument will be replaced with tip hash I think, unless `standaloneContext.BlockChain` does not exist.

Will it have misbehavior still?
(Though agree with allowing nullable `blockHash` doesn't seems to be good)

Reply:
In detail, it's the problem that `StandaloneContext.BlockChain` is nullable. 🤔 At this time, this conversation can be resolved and the Libplanet-side problem (i.e., `BlockChain.GetBlockState(BlockHash?)`) should be continued.